### ISSUE TITLE
I am making a BMW Logo

### ISSUE THREAD

**Issue by Harshit-Pathak** [salil-naik/css-art-gallery#59]

**Comment by sakshigupta0119** [comment]
I am interested in this please assign me this issue.

**Comment by salil-naik** [comment]
@Harshit-Pathak, are you still working on this issue? If not, let me know so that I assign it to someone else.

**Comment by Harshit-Pathak** [comment]
@salil-naik Yes sir I am working but I am getting an error on running yarn start command, also I have yarn installed but still its showing so I am stuck
